packages/lesswrong/components/sequences/SequencesPage.tsx

```diff
@@ -73,8 +75,8 @@ const styles = (theme: ThemeType): JssStyles => ({
       marginTop: -100,
     },
     [theme.breakpoints.down('xs')]: {
-      marginTop: theme.spacing.unit,
-      padding: theme.spacing.unit
+      marginTop: isEAForum ? undefined : theme.spacing.unit,
+      padding: isEAForum ? 16 : theme.spacing.unit
```

Comment:
I also slightly changed our mobile styling so that the sequence card has more padding and is higher up on the page / overlaps the banner a bit. I just thought it looked weird to have the huge banner taking up all the screen space on mobile.